pkg/util/iptables/iptables.go

```diff
@@ -631,7 +631,6 @@ const (
 	opFlushChain  operation = "-F"
 	opDeleteChain operation = "-X"
 	opListChain   operation = "-S"
-	opAppendRule  operation = "-A"
```

Comment:
I dont support removing it. Here we list all basic iptables operation command. They mostly looks like a group. And In fact, there are same operation(use "-A" otherwise const) in proxier.go of iptables. We can try to refactor them.